### ISSUE TITLE
SS4 support

### ISSUE THREAD

**Issue by pitchandtone** [heyday/silverstripe-optimisedimage#15]
Is anyone working on SS4 support and could give some timelines? Or provide a cost to get it upgraded, I have a motivated client.

**Comment by camspiers** [comment]
I'll take a look this weekend (US Time)

**Comment by 3Dgoo** [comment]
SilverStripe 4 support for this module would be great. Is there anything we can do to help with the upgrade of this?

**Comment by hp7** [comment]
Yes this would be great. Surprised it is not baked in to Silverstripe 4.1 UploadField already

**Comment by xini** [comment]
any updates on this?
